sound/soc/sof/intel/hda.c

```diff
@@ -548,6 +548,57 @@ static const char *fixup_tplg_name(struct snd_sof_dev *sdev,
 	return tplg_filename;
 }
 
+static int dmic_topology_fixup(struct snd_sof_dev *sdev,
+			       const char **tplg_filename,
+			       const char *idisp_str,
+			       int *dmic_found)
+{
+	const char *dmic_str;
+	const char *default_tplg_filename = *tplg_filename;
```

Comment:
can we move  "const char *dmic_str; " after "const char *fixed_tplg_filename" to reform the shape of declaration ?

Reply:
I'll do it with a fixup to avoid another round of reviews